pkgs/top-level/all-packages.nix

```diff
@@ -3365,6 +3365,10 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
+  aoc-cli = callPackage ../tools/misc/aoc-cli { 
+    inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
According to the new oh my gosh, (inheritance in all-packages is now discouraged]https://github.com/NixOS/nixpkgs/issues/204303).

Reply:
I had it without the inherit originally but was told to switch it (see above).

Reply:
You can remove this line and add

```nix
let
  inherit (darwin.apple_sdk.frameworks) Security;
in
stdenv.mkDerivation ...
```

Reply:
Ok, that is what it was originally before a few people told me to switch it. Just want to make sure 100% this is what I should do as I don't want to keep changing it.

Reply:
OK, I will ignore this and go ahead. I am far from being happy on this dark magic of splicing attributes.